packages/fstar/fstar.0.9.0/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "protz@microsoft.com"
+homepage: "http://fstar-lang.org"
+license: "Apache"
+depends: [
+  "ocamlfind"
+  "batteries"
```

Comment:
@nikswamy IIRC the philosophy of the opam-repository is to have open bounds to avoid getting stuck with old packages for no reasons.